src/components/navigation-header/components/product-page-content/utils/get-nav-items.ts

```diff
@@ -61,15 +61,29 @@ export function getNavItems(currentProduct: ProductData): NavItem[] {
 		}
 	}
 
+	let docsNavItems: NavItem[]
+	if (currentProduct.slug === 'terraform') {
```

Comment:
We should add some breadcrumb notes around why terraform is an outlier.

Reply:
🙌 Great call! Updated with some breadcrumbs on the "Terraform has lots of docs categories, the top nav bar has limited space" issue.